test/DynamoCoreWpfTests/NodeViewTests.cs

```diff
@@ -386,7 +388,9 @@ public void NodeSuggestions_InputPortBuiltInNode_AreCorrect()
             type = port.GetInputPortType();
             Assert.AreEqual("string", type);
 
-            var suggestions = inPorts[1].GetMatchingNodes();
+            var searchViewMode = (ViewModel.CurrentSpaceViewModel.NodeAutoCompleteSearchViewModel as NodeAutoCompleteSearchViewModel);
+            searchViewMode.PortViewModel = inPorts[1];
+            var suggestions = searchViewMode.GetMatchingNodes();
```

Comment:
@aparajit-pratap And team, curious about your thoughts here. IMHO, the reasons I would like to move this API is below
1. The API is about PortView logic
2. The API is getting `NodeSearchElement` which is more tightly connected to search view model.
3, We may even want to rename the API based on the reason above.
4. Still unit testable this way

Reply:
I'm okay either way. I don't feel strongly about moving it if it's mainly for the reason that it returns `NodeSearchElement`. We could also make it return plain strings or something else and wrap them into `NodeSearchElement`s inside of `PopulateAutocomplateCandidates` if that's the concern :)

Reply:
Maybe the latter statement I made is not so straightforward; it's easier to just return `NodeSearchElement` from the API so I'm okay with this change.

Reply:
On the opposite I think the current implementation is quite clean, but in the future since we may introduce ML module dependencies, I would rather not put this API under PortViewModel :)